ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/Cleaner.java

```diff
@@ -104,9 +103,8 @@ public class Cleaner extends MetaStoreCompactorThread {
   @Override
   public void init(AtomicBoolean stop) throws Exception {
     super.init(stop);
+    checkInterval = conf.getTimeVar(HiveConf.ConfVars.HIVE_COMPACTOR_CLEANER_RUN_INTERVAL, TimeUnit.MILLISECONDS);
```

Comment:
why not just in place rename?

Reply:
The declaration of the variable is moved to the base class, as all Compactor threads had concept of checkInterval.

Reply:
i get it, but why to reorder the code if you could just rename in-place

Reply:
Got your point, will fix the order back to the original.